API/DTOs/AggregatePlayerMatchStatsDTO.cs

```diff
@@ -15,21 +15,6 @@ public class AggregatePlayerMatchStatsDTO
     /// </summary>
     public double HighestRating { get; set; }
 
-    /// <summary>
-    /// The peak global rank achieved by the player during the period
-    /// </summary>
-    public int HighestGlobalRank { get; set; }
-
-    /// <summary>
-    /// The peak country rank achieved by the player during the period
-    /// </summary>
-    public int HighestCountryRank { get; set; }
-
-    /// <summary>
-    /// The peak rating percentile achieved by the player during the period
-    /// </summary>
-    public double HighestPercentile { get; set; }
```

Comment:
Actually no, since `AggregatePlayerMatchStatsDTO` is meant to represent stats over a certain time period and that proposal wouldn't be storing any historical data. These should stay removed in this case.